student.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Student < Person
+  def initialize(age, classroom, name, parent_permission)
```

Comment:
- [ ] the `name` attribute should be set to `"Unknown"` and the `parent_permission` should be set to `true`